tests/Integration/all_integration_server_tests.sh

```diff
@@ -1,100 +1,106 @@
 #!/usr/bin/env bash
 #-------------------------------------------------------------------------------
 # A convenient way to run all the integration tests for servers
-#
-# It supposes that DIRAC is installed in ${SERVERINSTALLDIR}
 #-------------------------------------------------------------------------------
 
 echo -e '****************************************'
 echo -e '*******' "integration server tests" '*******\n'
 
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
why changing the base dir ?

Reply:
It becomes necessary for the Python 3 installation layout where `ServerInstallDIR/DIRAC` doesn't exist. It's not strictly needed here but it will cause fewer conflicts if I include it now.

Reply:
The reason for having `ServerInstallDIR` and `ClientInstallDIR` (and `PilotInstallDIR`) was for installing a DIRAC server and a (different) DIRAC client on the same box, just in different directories. Would it still be possible to install different versions of server and client on the same box?

Reply:
You can still do that, you just need to run `all_integration_server_tests.sh` from the same location as the one you want to get the tests from.

Reply:
actually, how would you install two different versions ?

Reply:
I think it isn't quite supported by the integration tests at the moment. In #4995 it will be easier as part of my factorising the tests into a Python script to make them easier to run locally:

```bash
$ ./integration_tests.py --help
Usage: integration_tests.py [OPTIONS] COMMAND [ARGS]...

  Run the DIRAC integration tests.

  A local DIRAC setup can be created and tested by running:

    ./integration_tests.py create

  This is equivalent to running:

    ./integration_tests.py prepare-environment
    ./integration_tests.py install-server
    ./integration_tests.py install-client
    ./integration_tests.py test-server
    ./integration_tests.py test-client

  The test setup can be shutdown using:

    ./integration_tests.py destroy

  To run with a non-standard configuration add feature flags to
  "create" or "prepare-environment", e.g:

    ./integration_tests.py prepare-environment \
        CLIENT_USE_PYTHON3=Yes \
        SERVER_USE_PYTHON3=Yes \
        MYSQL_VER=8.0

  See below for additional subcommands which are useful during local
  development.

  ## Extensions

  TODO

  ## Command completion

  Command competion of typer based scripts can be enabled by running:

    typer --install-completion

  After restarting your terminal you command completion is available using:

    typer ./integration_tests.py run ...

Options:
  --install-completion  Install completion for the current shell.
  --show-completion     Show completion for the current shell, to copy it or
                        customize the installation.

  --help                Show this message and exit.

Commands:
  create               Start a local instance of the integration tests
  destroy              Destroy a local instance of the integration tests
  exec-client          Start an interactive session in the client container.
  exec-server          Start an interactive session in the server container.
  install-client
  install-server
  list-services        List the services which have been running.
  logs                 Show DIRAC's logs from the service container.
  prepare-environment
  runsvctrl            Execute runsvctrl inside the server container.
  test-client          Run the client integration tests.
  test-server          Run the server integration tests.
```